src/agent/src/rpc.rs

```diff
@@ -915,6 +918,13 @@ impl agent_ttrpc::AgentService for AgentService {
             "empty update interface request",
         )?;
 
+        let pcipath = pci::Path::from_str(&interface.pciPath)
+            .map_ttrpc_err(|e| format!("Unexpected pci-path for network interface: {:?}", e))?;
+
+        wait_for_net_interface(&self.sandbox, &pcipath)
+            .await
+            .map_ttrpc_err(|e| format!("interface not available: {:?}", e))?;
+
```

Comment:
@amshinde AFAIK, currently we support both virtio-pci and virtio-mmio bus in kata. this is for virtio-pci but there's no pci path existing in virtio-mmio bus, we need take this into account, right ?

Reply:
And as [the comment](https://github.com/kata-containers/kata-containers/pull/9895#issuecomment-2190005113) said, it seems not work for net device with vfio passthrough.

Reply:
@Apokleos Ack, I tested this with qemu and clh where pci bus is in use. I will special case this to pci case only. Will have to deal with virtio-mmio separately, maybe I can address that in a follow-up PR.